cmd/notary-server/main.go

```diff
@@ -32,6 +23,10 @@ import (
 const (
 	jsonLogFormat = "json"
 	DebugAddress  = "localhost:8080"
+	// This is the generally recommended maximum age for Cache-Control headers
+	// (one year, in seconds, since one year is forever in terms of internet
+	// content)
+	maxMaxAge = 60 * 60 * 24 * 365
```

Comment:
I don't feel too strongly about this constant either since it's only used here, when we use it in another place we can move it to `const.go`